service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfChangeControllerTest.java

```diff
@@ -67,129 +56,256 @@
+                .firstName("Other")
+                .lastName("Solicitor")
+                .email(solicitorEmail)
+                .organisation(Organisation.builder().organisationID("123").build()).build())
             .build();
 
-        final CaseData caseData = CaseData.builder()
+        final CaseData caseDataBefore = CaseData.builder()
+            .id(CASE_ID)
+            .caseName(caseName)
+            .respondents1(wrapElements(
+                otherRespondent,
+                Respondent.builder()
+                .legalRepresentation("Yes")
+                .solicitor(RespondentSolicitor.builder()
+                    .firstName("Old")
+                    .lastName("Solicitor")
+                    .email(solicitorEmail)
+                    .organisation(Organisation.builder().organisationID("123").build()).build())
+                .build()
+            )).build();
+
+        private final CaseData caseData = CaseData.builder()
             .id(CASE_ID)
-            .changeOrganisationRequestField(changeRequest)
-            .respondents1(List.of(respondent1, respondent2, respondent3))
+            .caseName(caseName)
+            .respondents1(wrapElements(
+                otherRespondent,
+                Respondent.builder()
+                .legalRepresentation("Yes")
+                .solicitor(RespondentSolicitor.builder()
+                    .firstName("New")
+                    .lastName("Solicitor")
+                    .email(solicitorEmail)
+                    .organisation(Organisation.builder().organisationID("123").build()).build())
+                .build()
+            )).build();
+
+        CallbackRequest callbackRequest = CallbackRequest.builder()
+            .caseDetailsBefore(asCaseDetails(caseDataBefore))
+            .caseDetails(asCaseDetails(caseData))
             .build();
 
-
-        final AboutToStartOrSubmitCallbackResponse actualResponse = postAboutToStartEvent(caseData);
-
-        final CaseData updatedCaseData = extractCaseData(requestCaptor.getValue());
-
-        final Element<Respondent> expectedRespondent = update(respondent1, SOLICITOR_USER, NEW_ORGANISATION);
-
-        assertThat(updatedCaseData.getRespondents1()).containsExactly(expectedRespondent, respondent2, respondent3);
-        assertThat(actualResponse).isEqualTo(ASSIGNMENT_RESPONSE);
-    }
-
-    private Element<Respondent> update(Element<Respondent> respondent, UserDetails solicitor, Organisation org) {
-        return element(respondent.getId(), respondent.getValue().toBuilder()
-            .legalRepresentation("Yes")
-            .solicitor(RespondentSolicitor.builder()
-                .firstName(solicitor.getForename())
-                .lastName(solicitor.getSurname().orElse(null))
-                .email(solicitor.getEmail())
-                .organisation(org)
-                .build())
-            .build());
+        @Test
+        void shouldNotifyNewRespondentSolicitorWhenCaseIsSubmitted() {
```

Comment:
this is integration test and not unit test, you want to check scenario, so with given data I would keep single test that do assertions on both emails